### PR TITLE
Setting is not in SP Admin Center

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3482

### CODE REVIEW

File: SharePoint/SharePointOnline/view-search-usage-reports-modern-sites.md [hunk 1]
```diff
@@ -32,7 +32,6 @@ If you're a SharePoint Administrator in your organization, you're probably asked
 - Which queries are getting low clicks as they're not showing up in any results?
 - How often are query rules firing and how often are people clicking promoted results?
 
-This article describes how you can use search reports in the <a href="https://go.microsoft.com/fwlink/?linkid=2185219" target="_blank">SharePoint admin center</a> to find answers to these questions.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/view-search-usage-reports-modern-sites.md:35]:
```suggestion
```

**Comment by scanum** [comment]
@get-itips

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by get-itips** [comment]
@Lovina-Saldanha please review and merge thanks

**Comment by get-itips** [comment]
@tiburd Tina, can you help me here? Thanks

**Comment by get-itips** [comment]
@Lovina-Saldanha has approved by email, can we merge? @tiburd

**Pull request merged**